.teamcity/src/main/kotlin/configurations/FunctionalTest.kt

```diff
@@ -66,6 +66,9 @@ class FunctionalTest(
     ).filter { it.isNotBlank() }.joinToString(separator = " ")
 
     if (parallelizationMethod is ParallelizationMethod.TeamCityParallelTests && parallelizationMethod.numberOfBatches > 1) {
+        params {
+            param("env.PARALLEL_TESTS_ENABLED", "1")
```

Comment:
```suggestion
            param("env.TEAMCITY_PARALLEL_TESTS_ENABLED", "1")
```

I wonder if we should prefix this env-var, given that parallel tests is pretty generic.